### ISSUE TITLE
Add Function AtTime(time.Time)

### ISSUE THREAD

**Issue by lookfirst** [procyon-projects/chrono#5]

**Comment by lookfirst** [comment]
Obviously, need another method name. You're taking the 'proposal' too literal.

**Comment by burakkoken** [comment]
For now, this change will not be made. that's why closing the issue.

**Comment by lookfirst** [comment]
The issue is still perfectly valid though. You haven't resolved it at all. It is pretty silly to have to spell out each field. Just add another method to simplify life for people. Or not... it seems like you're not updating this library at all anyway.

**Comment by burakkoken** [comment]
Yeap, this is valid issue but we cannot change the existing function signature. that's why I had close this this issue. however this issue actually arise again, we can add a function named AtTime.

**Comment by lookfirst** [comment]
@burakkoken 166 days, not bad. Thanks!
